### ISSUE TITLE
Editor styles are missing in production files

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#484]
## Issue Overview

When you run `npm run bundle`, you should have have `editor-styles-min.css` in `assets/css/editor/`. This file is missing.

**Comment by benoitchantre** [comment]
@ataylorme One solution is to add the path to `export` in `config.default.json`, but there might be a better way to fix that.

```json
"export": {
  "compress": true,
  "generatePotFile": true,
  "filesToCopy": [
    "LICENSE",
    "readme.txt",
    "style.css",
    "screenshot.png",
    "assets/css/editor/editor-styles.min.css"
  ]
}
```

**Comment by ataylorme** [comment]
Closed by #495
